### PR TITLE
Create README.tr.md

### PR DESCRIPTION
**Pull request by ayyucedemirbas**
README.md in Turkish

### CODE REVIEW

File: README.tr.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Apollo-11
+[![NASA][1]][2]
+
+:crossed_flags:
+[Bahasa Indonesia][ID],
+[Català][CA],
+[Deutsch][DE],
+**Türkçe**,
```
Comment (wopian) [inline README.tr.md]: Please put this in alphabetical order

Comment (ayyucedemirbas) [reply README.tr.md]: Solved

**Comment by sariug** [comment]
#423 seems a bit better I guess.

**Comment by btargac** [comment]
#423 covers more files and the language translation is a bit better.

**Pull request closed without merging**